packages/web-service/src/pages/common/check-answers.njk

```diff
@@ -0,0 +1,16 @@
+{% extends "information-page.njk" %}
+{% from "button/macro.njk" import govukButton %}
+{% set widthClass = 'govuk-grid-column-two-thirds' %}
+{% set title = 'Check your answers' %}
+{% from "answer-summary.njk" import answerSummary %}
+
+{% block pageContent %}
+    {{ answerSummary(questionMap, checkData) }}
+    {{ govukButton({
+        attributes: { id: 'continue' },
+        preventDoubleClick: true,
+        name: 'continue',
+        text: buttonText if buttonText else 'Confirm',
+        classes: "govuk-!-margin-top-5"
+    }) }}
+{% endblock %}
```

Comment:
There's a few of these missing new lines at the end of files - I suggest checking your linting rules are being applied properly in your IDE

Reply:
I will take a look. Its only the templates effected.